### PR TITLE
[plugin.video.nhlgcl@jarvis] 2021.10.1

### PR DESCRIPTION
**Pull request by eracknaphobia**
### Add-on details:

- **General**
  - Add-on name: NHL TV™
  - Add-on ID: plugin.video.nhlgcl
  - Version number: 2021.10.1
  - Kodi/repository version: jarvis

- **Code location**
  - URL: https://github.com/eracknaphobia/plugin.video.nhlgcl
  
With NHL.TV you can access revolutionary 60fps (frames per second) video through Kodi
        

### Description of changes:

            - Add Seattle to Favorite Team selection
            - Refactor Fav Team code
            - Build game images dynamically
        

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.nhlgcl/resources/lib/globals.py [hunk 1]
```diff
@@ -318,60 +322,63 @@ def getFavTeamId():
...
-    '''
-    Hold off until PIL is fixed for Android OS
-    try:
-        createGameIcon(home,away,image_path)
-    except:
-        image_path = ICON
-        pass
-    '''
+    if not os.path.exists(folder_path):
+        os.makedirs(folder_path)
```
Comment (basrieter) [inline plugin.video.nhlgcl/resources/lib/globals.py:331]: You are not allowed to modify anything in the Kodi add-on folder. Not even your own add-on files. 

The best solution would be to store the images in the profile folder under `userdata\addon_data\plugin.video.nhlgcl`

**Review by basrieter — CHANGES_REQUESTED** [review]
Please review my comment regarding modifying the add-on folders. (I do like the fact you are actually loading the files online instead of including them).

After that is fixed, I will review the rest.

File: plugin.video.nhlgcl/resources/lib/globals.py (hunk 1 above)
Comment (eracknaphobia) [reply plugin.video.nhlgcl/resources/lib/globals.py:331]: Ok, it should save the images in the addons user_data folder now.

Comment (eracknaphobia) [reply plugin.video.nhlgcl/resources/lib/globals.py:331]: Any chance this one can be reviewed? The requested changes have been made.

**Review by enen92 — APPROVED** [review]

**Pull request merged**